### PR TITLE
Add TDA feature flag

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

Add TDA feature flag.

More TDA info, check the [ADR](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/adr/0021-tda-applications.md)

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[TDA Publish: Add Feature flag](https://trello.com/c/qscbr6bk/1459-tda-publish-add-feature-flag)

File: config/settings.yml [hunk 1]
```diff
@@ -92,6 +92,7 @@ feedback:
   link: https://forms.office.com/pages/responsepage.aspx?id=yXfS-grGoU2187O4s0qC-SKECobyE75EtuJMp8rYxZtURTNaTTJaTVhBQlQzM1RESTJDVlBERk1JQS4u
 
 features:
+  tda: true
```
[the remark this answers is not in the document]
Comment (tomas-stefano) [reply config/settings.yml:95]: Sure :)

[the remark this answers is not in the document]
Comment (tomas-stefano) [reply config/settings.yml]: Done!

**Review by avinhurry — APPROVED** [review]
:shipit:

**Pull request merged**